libraries/Microsoft.Bot.Builder.Prompts/Microsoft.Bot.Builder.Prompts.csproj

```diff
@@ -48,10 +48,14 @@
 		<PackageReference Include="Microsoft.Bot.Schema" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
 		<PackageReference Include="Microsoft.Bot.Schema" Condition=" '$(PackageVersion)' != '' " Version="$(PackageVersion)" />
```

Comment:
Here is an example of how to have nuget package reference which works for local builds and for signed versioned.  All of the project files that refer between each other should use this pattern. Can you please update the .AI projects?